src/pkg/backup/backup.go

```diff
@@ -31,6 +33,9 @@ type Backup struct {
 	// Selector used in this operation
 	Selector selectors.Selector `json:"selectors"`
 
+	// Version represents the version of the backup format
```

Comment:
Since this is a change in the backup model should we bump up [`BaseModel.Version`](https://github.com/alcionai/corso/blob/e35c6cd17eaaf0251f21271bba6e4d3089dd61bf/src/internal/model/model.go#L58-L60)?
@ashmrtn @ryanfkeepers @vkamra